### PR TITLE
New Files Added: Assignment1, assignment2, assignment3 and assignment4

### PR DESCRIPTION
**Pull request by Pranjali1999**

### CODE REVIEW

**Review by Akash-ing — APPROVED** [review]

**Review by Bhavika222 — APPROVED** [review]

**Review by vishnukopnar — APPROVED** [review]

**Review by vinod2406 — APPROVED** [review]

**Review by akash19972306 — APPROVED** [review]

File: Pranjali/HTML/Assi1/Assi4.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    <head>
+        <title>INTRODUCT/title>
+    </head>
+    <body>
+        <h1>Hello!</h1>
+        <p>My name is 
+            <b>Pranjali Ajmire.</b></p><br>  
+        <p>I am a Software Engineer from <b>JDIET</b></p> <br>       
+       <p>I did my graduation from <b>SGBSU</b>University</p>
+
```
Comment (Pawan1005) [inline Pranjali/HTML/Assi1/Assi4.html]: remove extra lines

File: Pranjali/HTML/Assi2/Assignment.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            div{
+                background-color: rgb(202, 121, 105);
+            }
+            mark{
+                background-color: rgb(248, 248, 54);
+            }
+            mark1{
+                background-color: blue;
+            }
+
```
Comment (Pawan1005) [inline Pranjali/HTML/Assi2/Assignment.html]: remove extra line

File: Pranjali/HTML/Assi2/Assignment.html [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+            individual pages on the <a href="https://www.WorldWide Web.com">World Wide Web.</a> </p> 
+        <hr>
+        <p>They called this search engine<i>Backrub.</i></p>
+
+        <h3>Initial public offering</h3>
+        <p>Google's initial public offering (IPO) took place five years later, on 19th August 2004.<br>
+            At that time Larry Page, Sergey Brin, and Eric Schmidt agreed to work together at Google for <del>30</del> <ins>20</ins><br>
+            years, until the year 2024.</p>
+        <p>Thank You!!</p> 
+
```
Comment (Pawan1005) [inline Pranjali/HTML/Assi2/Assignment.html]: remove extra line

File: Pranjali/HTML/Assi2/Assignment.html [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        <h3>Initial public offering</h3>
+        <p>Google's initial public offering (IPO) took place five years later, on 19th August 2004.<br>
+            At that time Larry Page, Sergey Brin, and Eric Schmidt agreed to work together at Google for 30 <u>20</u><br>
+            years, until the year 2024.</p>
+        
+        <h4 style="color:rgb(26, 73, 26);">Thank You!!</h4> 
+      </div>  
+        
+       
+
```
Comment (Pawan1005) [inline Pranjali/HTML/Assi2/Assignment.html]: remove extra line

File: Pranjali/HTML/Assi3/assi3_2.html [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+                This is the homepage subheading.This is the homepage subheading.
+                This is the homepage subheading.This is the homepage subheading.
+                This is the homepage subheading.This is the homepage subheading.
+                This is the homepage subheading.This is the homepage subheading.
+                This is the homepage subheading.This is the homepage subheading.
+            </p3>
+        </div>
+        
+    </body>
+</html>
```
Comment (Pawan1005) [inline Pranjali/HTML/Assi3/assi3_2.html]: add new line here

File: Pranjali/HTML/Assi3/assi3_3.html [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+        <ol>
+            <li>Bipin</li>
+            <li>Priyanka</li>
+            <li>Yashasvi</li>
+        </ol>
+
+
+
+    </body>
+</html>
```
Comment (Pawan1005) [inline Pranjali/HTML/Assi3/assi3_3.html]: add new line

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
add new line at the end of file 
and remove extra spaces

File: Pranjali/HTML/Assi3/assi3_1.html [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+            }
+            div1{
+                color: rgb(35, 31, 114);
+            }
+        </style>
+        
+    </head>
+    <body>
+        <h1><p class="p1">Ambersoft</p></h1>
+
```
Comment (preetipie22) [inline Pranjali/HTML/Assi3/assi3_1.html]: remove extra lines in other codes as well

File: Pranjali/HTML/Assi3/assi3_1.html [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+                <li>Monoj</li>
+            </ul>  
+          </ol><br>
+        </div>
+
+
+
+
+
+        <div1>
```
Comment (preetipie22) [inline Pranjali/HTML/Assi3/assi3_1.html]: Hey, @Pranjali1999  there is no tag `<div1>` , instead use `<div>`. Use id if you want to distinguish them

File: Pranjali/HTML/Assi3/assi3_1.html [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+                color: rgb(35, 31, 114);
+            }
+        </style>
+        
+    </head>
+    <body>
+        <h1><p class="p1">Ambersoft</p></h1>
+
+        <div>
+        <ol class="d"><num>1.</num>MI-MEAN
```
Comment (preetipie22) [inline Pranjali/HTML/Assi3/assi3_1.html]: there is no such tag like `<num>`

File: Pranjali/HTML/Assi3/assi3_1.html [hunk 10]
```diff
@@ -0,0 +1,89 @@
+<html>
+    <head>
+        <style>
+            .P1 {
```
Comment (preetipie22) [inline Pranjali/HTML/Assi3/assi3_1.html]: class names are case-sensitive

File: Pranjali/HTML/Assi3/assi3_1.html [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+                font-family: "Times New Roman", 'Times New Roman', Times, serif;
+            }
+            ul.a{
```
Comment (preetipie22) [inline Pranjali/HTML/Assi3/assi3_1.html:9]: you can give the type in html code as well

File: Pranjali/HTML/Assi3/assi3_2.html [hunk 12]
```diff
@@ -0,0 +1,113 @@
...
+        <div>
+            <img src="img.jpeg" style="height: 400; width: 300;">
+        </div>
+
+        <div class="top-right">
+            <img src="img1.jpeg" style="height: 400; width: 300;">
+        </div>
+
+        <div1 class="centered">
+            <p2>
```
Comment (preetipie22) [inline Pranjali/HTML/Assi3/assi3_2.html]: there is no tag like `<p2>`, `<div>`, use class or id instead. Check this for other codes as well

File: Pranjali/HTML/Assi1/Assi4.html [hunk 13]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>INTRODUCT/title>
```
Comment (preetipie22) [inline Pranjali/HTML/Assi1/Assi4.html]: `<title>` tag is not closed

File: Pranjali/HTML/Assi3/assi3_1.html [hunk 14]
```diff
@@ -0,0 +1,89 @@
...
+                color: rgb(51, 19, 19);
+            }
+            div1{
+                color: rgb(35, 31, 114);
+            }
+        </style>
+        
+    </head>
+    <body>
+        <h1><p class="p1">Ambersoft</p></h1>
```
Comment (preetipie22) [inline Pranjali/HTML/Assi3/assi3_1.html]: class names are case sensitive

File: Pranjali/HTML/Assi4/assi4_2.html [hunk 15]
```diff
@@ -0,0 +1,89 @@
...
+            
+        </style>
+    </head>
+
+    <body>
+        <h2 style="color: rgb(155, 67, 155);"><center>Jan's Spring Term Afternoon Schedule</center></h2>
+        
+        <table style="width:100%;">
+            <tr class="a">
+                <th>      </th>
```
Comment (preetipie22) [inline Pranjali/HTML/Assi4/assi4_2.html]: no need to give space in other places as well

File: Pranjali/HTML/Assi4/assi4_2.html [hunk 16]
```diff
@@ -0,0 +1,89 @@
...
+                border: 2px solid black;
+                border-right: 10px solid grey;
+            }
+
+            
+        </style>
+    </head>
+
+    <body>
+        <h2 style="color: rgb(155, 67, 155);"><center>Jan's Spring Term Afternoon Schedule</center></h2>
```
Comment (preetipie22) [inline Pranjali/HTML/Assi4/assi4_2.html]: `<center>` is not supported in  HTML5

**Review by preetipie22 — CHANGES_REQUESTED** [review]
Please make the changes

**Comment by avinash0070** [comment]
Pranjali1999 if you done with your all changes the resolve all comments

**Comment by avinash0070** [comment]
Need proper title for this pull request
and also assign yourself
done all changes as soon as possible

File: Pranjali/HTML/Assi2/Assignment.html [hunk 17]
```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <style>
+            h1 {
+                text-align: center;
+            }
```
Comment (avinash0070) [inline Pranjali/HTML/Assi2/Assignment.html:8]: add new line

File: Pranjali/HTML/Assi2/Assignment.html [hunk 18]
```diff
@@ -0,0 +1,78 @@
...
+            h2 {
+                text-align: center;
+            }
```
Comment (avinash0070) [inline Pranjali/HTML/Assi2/Assignment.html:12]: add new line

File: Pranjali/HTML/Assi2/Assignment.html [hunk 19]
```diff
@@ -0,0 +1,78 @@
...
+            h4 {
+                text-align: center;
+            }
+            div {
+                background-color: rgb(202, 121, 105);
+            }
+            mark {
+                background-color: rgb(248, 248, 54);
+            }
```
Comment (avinash0070) [inline Pranjali/HTML/Assi2/Assignment.html:24]: add new line

File: Pranjali/HTML/Assi2/Assignment.html [hunk 20]
```diff
@@ -0,0 +1,78 @@
...
+            h2 {
+                text-align: center;
+            }
+            h4 {
+                text-align: center;
+            }
+            div {
+                background-color: rgb(202, 121, 105);
+            }
```
Comment (avinash0070) [inline Pranjali/HTML/Assi2/Assignment.html:20]: add new line

File: Pranjali/HTML/Assi2/Assignment.html [hunk 21]
```diff
@@ -0,0 +1,78 @@
...
+        <style>
+            h1 {
+                text-align: center;
+            }
+            h2 {
+                text-align: center;
+            }
+            h4 {
+                text-align: center;
+            }
```
Comment (avinash0070) [inline Pranjali/HTML/Assi2/Assignment.html:16]: add new line

File: Pranjali/HTML/Assi3/assi3_2.html [hunk 22]
```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Pranjali/HTML/Assi3/assi3_2.html:1]: add new line at the end of all css property

File: Pranjali/HTML/Assi3/assi3_3.html [hunk 23]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Pranjali/HTML/Assi3/assi3_3.html:1]: add new line at the end of all css property

File: Pranjali/HTML/Assi3/assi3_4.html [hunk 24]
```diff
@@ -0,0 +1,51 @@
+<html>
+    <head>
+        <style>
+            h1 {
```
Comment (avinash0070) [inline Pranjali/HTML/Assi3/assi3_4.html:4]: add new line at the end of all css property

File: Pranjali/HTML/Assi4/assi4_1.html [hunk 25]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <style>
```
Comment (avinash0070) [inline Pranjali/HTML/Assi4/assi4_1.html:5]: add new line at the end of all css property

**Review by avinash0070 — CHANGES_REQUESTED** [review]

File: Pranjali/HTML/Assi2/Assignment.html [hunk 26]
```diff
@@ -0,0 +1,78 @@
...
+                <a href="https://www.World Wide Web.com">World Wide Web</a> .
+            </p>
+            <hr />
+            <p>
+                They called this search engine <i><mark class="q">Backrub.</mark></i>
+            </p>
+            <h3>Initial public offering</h3>
+            <p>
+                Google's initial public offering (IPO) took place five years later, on 19th August 2004.<br />
+                At that time Larry Page, Sergey Brin, and Eric Schmidt agreed to work together at Google for 30 <u>20</u><br />
```
Comment (preetipie22) [inline Pranjali/HTML/Assi2/Assignment.html]: You should not use <u> to underline text for presentation purposes, or to denote titles of books.
Instead use `<ins>`

File: Pranjali/HTML/Assi3/assi3_1.html [hunk 27]
```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <style>
+            .P1 {
```
Comment (preetipie22) [inline Pranjali/HTML/Assi3/assi3_1.html]: class names are case sensitive .p1

**Comment by avinash0070** [comment]
Change in title not in commit message 
 do this changes

**Review by preetipie22 — CHANGES_REQUESTED** [review]

**Review by avinash0070 — APPROVED** [review]

**Pull request merged**